Entities/ACrab.cpp

```diff
@@ -94,6 +94,11 @@ void ACrab::Clear()
     m_SweepTimer.Reset();
     m_PatrolTimer.Reset();
     m_JumpTimer.Reset();
+
+	// Dynamic AimRange -- MaximDude
```

Comment:
Please fix your indenting here, and the extra // at the bottom of this code. Also, I'd rather people don't put their names in code like that, it just feels weird. Git/sourcetree has a method to track exactly who changed each line and in what commit anyway, so there's no need for it.